src/Uno.Extensions.Configuration/Reloader.cs

```diff
@@ -32,10 +34,21 @@ public async Task ReloadAllFileConfigurationProviders(string? configFile = defau
 				}
 				else
 				{
-					var contents = File.ReadAllText(info.PhysicalPath);
-					if (Logger.IsEnabled(LogLevel.Debug)) Logger.LogDebugMessage($@"Contents '{contents}'");
-					if (Logger.IsEnabled(LogLevel.Debug)) Logger.LogDebugMessage($@"Loading from full path '{info.PhysicalPath}'");
-					fp.Load();
+					if (Logger.IsEnabled(LogLevel.Debug))
+					{
+						var contents = File.ReadAllText(info.PhysicalPath);
+						Logger.LogDebugMessage($@"Contents '{contents}'");
+						Logger.LogDebugMessage($@"Loading from full path '{info.PhysicalPath}'");
```

Comment:
This will become a performance issue, as the formatting is done regardless of the logging level